### PR TITLE
Created blogpost for Observability Game days at Skyscanner

### PR DESCRIPTION
**Pull request by jordibisbal8**
In this PR, we aim to explain how Skyscanner uses the open-telemetry-demo to train our engineers on their Observability skills.

Created issue: https://github.com/open-telemetry/opentelemetry.io/issues/4038

### CODE REVIEW

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (danielgblanco) [inline content/en/blog/2024/demo-skyscanner/index.md:4]: Looking at how blog titles are displayed on the side bar, would this title be potentially too long?

Screenshot 2024-02-22 at 14 45 42

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (danielgblanco) [inline content/en/blog/2024/demo-skyscanner/index.md:5]: I presume this is the title that appears on the side bar. Should it mention Skyscanner or the OTel Demo?

Comment (jordibisbal8) [reply content/en/blog/2024/demo-skyscanner/index.md]: I have updated the linkTitle :)

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (danielgblanco) [inline content/en/blog/2024/demo-skyscanner/index.md:5]: (nit) I think it's normally spelt OTel Demo :)

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:3]: Observability is always fun, no need to make it fun 😢 ... just kidding! great title :-)

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [reply content/en/blog/2024/demo-skyscanner/index.md:5]:
```suggestion
linkTitle: Skyscanner using OTel Demo
```

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:6]:
```suggestion
date: 2024-02-22
```
Keep this suggestion open until we figured out the publication date

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:11]:
```suggestion
At [Skyscanner](https://www.skyscanner.com), as in many organisations, teams tend to follow specific runbooks
```

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:36]:
```suggestion
[OpenTelemetry Demo](/docs/demo/),
```

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:38]:
```suggestion
Thanks to the [OpenTelemetry Protocol](/docs/specs/otlp/) (OTLP), it allows us to simply point the
```

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:41]: @open-telemetry/docs-approvers double checking if we are OK with naming an observability vendor or if this should be "to our chosen observability platform". I am OK with it since it's an end-user blog post.

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:47]:
```suggestion
ecosystem is open source, we can easily introduce any new features that will be
```

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:80]:
```suggestion
  taking the time to set this up and promoting it for us - Dominic Fraser
  (Senior Software Engineer)
```

Are those people OK with being named here, and if so, we can link their github accounts

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:83]:
```suggestion
  useful, as well as a lot of fun! :D - Polly Yankova (Software Engineer)
```

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:94]:
```suggestion
by browser traffic. To achieve this, we have contributed to the OpenTelemetry
Demo and enabled these features for other interested parties. For more
```

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:96]:
```suggestion
information, please have a look at the
[raised PR](https://github.com/open-telemetry/opentelemetry-demo/pull/1345).
```

makes the link a little bit more visibile.

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:66]: did you create this image or is it taken out of the book?

**Review by svrnm — CHANGES_REQUESTED** [review]
Hey, thanks for creating this blog post, what a great read!

cc @open-telemetry/demo-approvers @open-telemetry/demo-maintainers

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (jordibisbal8) [reply content/en/blog/2024/demo-skyscanner/index.md:66]: IIRC, I created it using an online tool where I was able to add the desired text :)

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (jordibisbal8) [reply content/en/blog/2024/demo-skyscanner/index.md:80]: Yes, they are more than fine :) One of them has and the other not, should we still link the one?

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [reply content/en/blog/2024/demo-skyscanner/index.md:66]: ok, that's all I wanted to know

**Review by reese-lee — APPROVED** [review]

**Review by mhausenblas — CHANGES_REQUESTED** [review]
Thanks a lot for contributing, we certainly need more of this content! One ask I have is to provide a bit more details on lessons learned or insights from participants around OpenTelemetry (what is hard, anything that stood out that we could improve, anything missing that we should add, etc.).

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (danielgblanco) [reply content/en/blog/2024/demo-skyscanner/index.md]: This is still a valid point (although outdated). Considering the trace screenshot below comes from New Relic, it makes me think it's okay to mention it here. I don't think the blogpost is trying to advertise New Relic features except OTLP ingest, and in any case it compares it to other platforms that also accept OTLP.

**Review by theletterf — APPROVED** [review]

**Review by svrnm — APPROVED** [review]

File: content/en/blog/2024/demo-skyscanner/index.md
Comment (svrnm) [inline content/en/blog/2024/demo-skyscanner/index.md:6]:
```suggestion
date: 2024-02-26
```

**Comment by jordibisbal8** [comment]
@svrnm before publishing this, could we wait until I receive the approval from a Skyscanner side today?

**Comment by svrnm** [comment]
> @svrnm before publishing this, could we wait until I receive the approval from a Skyscanner side today?

Sure! Thanks for the headsup! I updated the PR slightly such that it work, please pull those changes locally.

**Comment by jordibisbal8** [comment]
> > @svrnm before publishing this, could we wait until I receive the approval from a Skyscanner side today?
> 
> Sure! Thanks for the headsup! I updated the PR slightly such that it work, please pull those changes locally.

Approved from the Skyscanner side too :)

**Pull request merged**
